pkg/alerts/alertutils/alertutils.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/go-co-op/gocron"
 )
 
-type AlertType uint8 // type of the alert: Logs or Metrics
+type AlertType uint8 // type of the alert: Logs or Metrics or Minion
```

Comment:
nit: You don't really need this comment, since you define the enum values just below it, and we'd have to update this comment every time we add more options

Reply:
yes, I will remove it.